codalab/lib/worksheet_util.py

```diff
@@ -336,7 +336,7 @@ def is_file_genpath(genpath):
     return genpath.startswith('/')
 
 
-def interpret_genpath(bundle_info, genpath):
+def interpret_genpath(bundle_info, genpath, db_model=None, owner_cache=None):
     """
     Quickly interpret the genpaths (generalized path) that only require looking
```

Comment:
Document the new arguments.  Specifically, note what the input/output of `owner_cache` is.